src-java/kilda-persistence-tinkerpop/src/main/java/org/openkilda/persistence/ferma/frames/FlowEventDumpFrame.java

```diff
@@ -186,12 +186,20 @@
     public abstract void setReverseMeterId(MeterId reverseMeterId);
 
     @Override
-    @Property("group_id")
-    public abstract String getGroupId();
+    @Property("diverse_group_id")
```

Comment:
Does it mean that you wont be able to read stored group_id field in this release until the DB migration from the next release is done?

Reply:
Fixed.